src/Plugin/EloquaAppCloudContentResponderInterface.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Drupal\eloqua_app_cloud\Plugin;
+
+use Drupal\Component\Plugin\PluginInspectionInterface;
+
+/**
+ * Defines an interface for Eloqua AppCloud Content Responder plugins.
```

Comment:
Would be good to clarify that this is purely within the Eloqua admin UI rather than the actual e-mail (or landing page) that gets sent out / visited.

Reply:
Done.